source/utils/unhandle.ts

```diff
@@ -0,0 +1,34 @@
+import EventEmitter from 'events';
+
+type Origin = EventEmitter;
+type Event = string | symbol;
+type Fn = (...args: any[]) => void;
+
+interface Handler {
+	origin: Origin;
+	event: Event;
+	fn: Fn;
+}
+
+interface Unhandler {
+	once: (origin: Origin, event: Event, fn: Fn) => void;
+	unhandleAll: () => void;
+}
+
+export default (): Unhandler => {
```

Comment:
Can you add a doc code comment about the purpose of this and what problem it solves?